resources/testharness.js

```diff
@@ -1529,7 +1529,8 @@ policies and contribution forms [3].
         PASS:0,
         FAIL:1,
         TIMEOUT:2,
-        NOTRUN:3
+        NOTRUN:3,
+        SKIP:3
     };
```

Comment:
Is it intentional that there will not be a distinction between not run and skip?  Might this be desired behavior?

Reply:
Not intentional :)